example/90-shoot.yaml

```
@@ -402,6 +402,17 @@ spec:
     autoUpdate:
       kubernetesVersion: true
       machineImageVersion: true
+    autoRotation:
+      credentials:
+        observability:
+          enabled: true
+          rotationPeriod: 168h
+        sshKeypair:
+          enabled: true
+          rotationPeriod: 168h
+        etcdEncryptionKey:
+          enabled: true
+          rotationPeriod: 168h
```

These are all defaulted to false, so I think it's better to comment them and specify the default values:
```suggestion
  # autoRotation:
  #   credentials:
  #   observability:
  #     enabled: false
  #     rotationPeriod: 168h
  #   sshKeypair:
  #      enabled: false
  #      rotationPeriod: 168h
  #   etcdEncryptionKey:
  #      enabled: false
  #      rotationPeriod: 168h
```
(you might have to edit the indentation when you apply this comment 😅)